src/kixi/event_replay/inputstream_to_events.clj

```diff
@@ -0,0 +1,91 @@
+(ns kixi.event-replay.inputstream-to-events
+  (:require [gloss.io :as gloss.io]
+            [taoensso.nippy :as nippy])
+  (:import [java.nio ByteBuffer]
+           [java.io InputStream]))
+
+(def line-break 10)
+
+(def decode-buffer-size (int 1024))
+
+(defn input-stream->lines
```

Comment:
Worth looking at `lines-reduceable` here: https://tech.grammarly.com/blog/building-etl-pipelines-with-clojure ?

Reply:
That is a great blog post! Trouble with any line based stuff is it strings the contents, current stuff keeps it raw bytes until it gets to nippy thaw. Will check that out though, the line breaker is a bit messy.